erpnext/stock/report/stock_ledger/stock_ledger.js

```diff
@@ -75,7 +75,14 @@ frappe.query_reports["Stock Ledger"] = {
 			"fieldname":"project",
 			"label": __("Project"),
 			"fieldtype": "Link",
-			"options": "Project"
+			"options": "Project",
+			"hidden": 1
```

Comment:
- Why hide the project field ?

Reply:
Hi @hrwX please check out this link:
https://app.asana.com/0/1200414497004131/1200402551564277/f